sdk/search/azure-search/src/main/java/com/azure/search/DataSources.java

```diff
@@ -56,34 +56,37 @@ public static DataSource azureSql(String name, String sqlConnectionString, Strin
     /**
      * Creates a new DataSource to connect to an Azure SQL database.
      *
-     * @param name The name of the data source.
+     * @param dataSourceName The dataSourceName of the data source.
      * @param sqlConnectionString The connection string for the Azure SQL database.
-     * @param tableOrViewName The name of the table or view from which to read rows.
+     * @param tableOrViewName The dataSourceName of the table or view from which to read rows.
      * @return A new DataSource instance.
      */
-    public static DataSource azureSql(String name, String sqlConnectionString, String tableOrViewName) {
-        return DataSources.azureSql(name, sqlConnectionString, tableOrViewName, null, null, null);
+    public static DataSource createFromAzureSql(String dataSourceName, String sqlConnectionString,
+        String tableOrViewName) {
+        return DataSources.createFromAzureSql(dataSourceName, sqlConnectionString, tableOrViewName, null, null, null);
     }
 
     /**
      * Creates a new DataSource to connect to an Azure Blob container.
      *
-     * @param name The name of the data source.
-     * @param storageConnectionString The connection string for the Azure Storage account. It must follow this format:
-     * "DefaultEndpointsProtocol=https;AccountName=[your storage account]; AccountKey=[your account key];" Note that
-     * HTTPS is required.
-     * @param containerName The name of the container from which to read blobs.
-     * @param pathPrefix Optional. If specified, the data source will include only blobs with names starting with this
-     * prefix. This is useful when blobs are organized into "virtual folders", for example.
+     * @param dataSourceName The name of the data source.
+     * @param storageConnectionString The connection string for the Azure Storage account.
+     * It must follow this format: "DefaultEndpointsProtocol=https;AccountName=[your storage account];
+     * AccountKey=[your account key];" Note that HTTPS is required.
```

Comment:
Thoughts on moving the explanation into the general Javadoc section? Also, let's make sure proper Javadoc formatting is used here.

```java
The Storage connection string must use this format:
<p>
{@code "DefaultEndpointsProtocol=https;AccountName=[your storage account];AccountKey=[your account key]:}
<p>
<em>Note: The connection string must use HTTPS.<em>
```

Reply:
Agreed upon the javadoc format. 
I would rather to leave it in parameter. User can find the usage of the connection string from param directly instead of searching on method description. It is more user friendly to me.